views/editor/nav-options.html

```diff
@@ -93,6 +93,13 @@
                 <div class="toggle-button"></div>
               </div>
             </li>
+            <li>
+              <span>{{ gettext("XML/SVG") }}</span>
```

Comment:
No, *specifically* the `en-US` one only. That's the master file. You don't edit any others, that is handled through by the localizers.